src/aleph/vm/orchestrator/views/operator.py

```diff
@@ -170,7 +170,7 @@ async def operate_reboot(request: web.Request, authenticated_sender: str) -> web
     if execution.is_running:
         logger.info(f"Rebooting {execution.vm_hash}")
         if execution.persistent:
-            await pool.systemd_manager.restart(execution.controller_service)
+            pool.systemd_manager.restart(execution.controller_service)
```

Comment:
I see that the underlying function calls `dbus.Interface(...).RestartUnit(service, "replace")`.

Is that call IO limited by communicating via DBus in a synchronous manner ? In that case, we should open another issue to make the methods of `SystemDManager` asynchronous and prevent them from blocking the entire orchestrator.

Reply:
Yes, we can open issues and improve it later to avoid block the entire orchestrator loop.

Reply:
https://app.clickup.com/t/86bxjfcp2